docs/standard/native-interop/best-practices.md

```diff
@@ -112,9 +122,9 @@ public struct UnicodeCharStruct
 }
 ```
 
-`string` is blittable if it isn't contained in another type and it's being passed as an argument that is marked with `[MarshalAs(UnmanagedType.LPWStr)]` or the `[DllImport]` has `CharSet = CharSet.Unicode` set.
+`string` contains blittable contents if it isn't contained in another type and it's being passed as an argument that is marked with `[MarshalAs(UnmanagedType.LPWStr)]` or the `[DllImport]` has `CharSet = CharSet.Unicode` set.
 
-You can see if a type is blittable by attempting to create a pinned `GCHandle`. If the type isn't a string or considered blittable, `GCHandle.Alloc` will throw an `ArgumentException`.
+You can see if a type is blittable or contains blittable contents by attempting to create a pinned `GCHandle`. If the type isn't a string or considered blittable, `GCHandle.Alloc` will throw an `ArgumentException`.
```

Comment:
I personally prefer the `Marshal.SizeOf()` method as it has less side effects. Unless that doesn't handle all the known cases.

Reply:
Marshal.SizeOf works for non-blittable value types as well so we can’t use it.